replica/replica.go

```diff
@@ -217,11 +217,20 @@ func construct(readonly bool, size, sectorSize int64, dir, head string, backingF
 	r.insertBackingFile()
 	r.ReplicaType = replicaType
 
+	return r, r.writeVolumeMetaData(true, r.info.Rebuilding)
```

Comment:
need to look at other callers of construct (clone? revert? etc) so that Preload is calling after this call